tests/cxx/unit_tests/ghostfragment/capping/DCLC.cpp

```diff
@@ -0,0 +1,63 @@
+#include "../test_ghostfragment.hpp"
+#include "DCLC.hpp"
+#include <ghostfragment/property_types/connectivity_table.hpp>
+#include <hydrocarbon/hydrocarbon_fragment.hpp>
+
+TEST_CASE("DCLC Capping") {
+
+    using namespace ghostfragment;
+    using namespace testing;
+
+    using the_pt       = pt::Capped;
+    using input_type   = chemist::FragmentedNuclei;
+    using result_type  = std::vector<chemist::CapSet>;
+    using conn_pt      = ghostfragment::ConnectivityTable;
+
+    auto mm        = initialize();
+    mm.change_submod("DCLC Capping", "Connectivity", 
```

Comment:
This should really be set in the `set_defaults` method defined in `src/ghostfragment/capping/capping.hpp`